### DISCUSSION TITLE
重启后不显示信号图标

### DISCUSSION THREAD

**Discussion by zhanghaoyu0** [MonwF/customiuizer#551] [Q&A]
更新 24—12—11 test 版本后重启状态栏不显示信号图标  （在米客内重启系统界面后才会显示）
lsp 日志  https://www.123865.com/s/9wuKVv-yu2k3
![Screenshot_2024-12-12-11-09-37-752_com.github.android.jpg](https://github.com/user-attachments/assets/24b78081-966c-499e-9d6c-2b26485aebd8)

**Comment by MonwF** [comment]
核心破解之类的模块关掉试试

**Reply by zhanghaoyu0** [reply]
关了重启还是不显示![Screenshot_2024-12-12-17-24-20-336_com.android.shell.jpg](https://github.com/user-attachments/assets/c23b39ef-bed7-47f9-81a7-3870d64b351d)

**Comment by MonwF** [comment]
换个lsp试试

**Reply by zhanghaoyu0** [reply]
好的  我试试看

**Reply by zhanghaoyu0** [reply]
我更新了 lsp 1.10.1 7157  还是不显示![Screenshot_2024-12-13-20-45-05-909_com.miui.home.jpg](https://github.com/user-attachments/assets/c0015ca9-4d93-413e-93b9-079c42c7f0d2)

**Reply by MonwF** [reply]
网盘里的米客调试版本呢

**Reply by zhanghaoyu0** [reply]
> 网盘里的米客调试版本呢

不行  有别人反馈同样的问题吗  我的手机是k70  hype os2.0  2.0.5.0

![Screenshot_2024-12-13-20-48-20-369_name.monwf.customiuizer.jpg](https://github.com/user-attachments/assets/90390418-fe35-47c9-ba2f-0264a3ee2323)

**Reply by zhanghaoyu0** [reply]
需要再上传一次 lsp 模块日志吗🤔

**Reply by MonwF** [reply]
> 需要再上传一次 lsp 模块日志吗🤔

不用 我没加任何调试信息 传也没用。。。除了双排，其他功能需要重启系统界面吗

**Reply by zhanghaoyu0** [reply]
其他暂时未发现异常

**Reply by MonwF** [reply]
问题大概是lsp服务通信失败 重连后导致的异常。。如果其他系统界面功能没问题就比较奇怪了，下次我开电脑有空加些日志再帮忙看下吧

**Reply by zhanghaoyu0** [reply]
好的

**Comment by MonwF** [comment]
下载网盘里调试专用的试试看

**Comment by MonwF** [comment]
已修复
